source/documentation/continuous-integration-and-delivery/secrets.md

```diff
@@ -0,0 +1,57 @@
+## Secrets
+
+The Reliability Engineering Concourse delegates secrets management to AWS
+Systems Manager Parameter Store and Key Management Service.
+
+Secrets can be managed by users with the `member` role using the [GDS
+CLI](https://github.com/alphagov/gds-cli):
+
+```
+gds cd secrets add
+```
+
+and
+
+```
+gds cd secrets remove
+```
+
+Concourse injects secrets into pipelines at runtime using double parentheses
+syntax:
+
+<pre><code>resources:
+  - name: my-repository
+    type: git
+    source:
+      uri: git@github.com:alphagov/my-repository.git
+      branch: master
+
+      # This is a secret
+      private_key: |
+        ((github-ssh-key))
+</code></pre>
+
+There are also some convenience variables provided for you:
+
+- `readonly_private_bucket_name` - the name of the private AWS S3 bucket
+- `readonly_private_bucket_arn` - the ARN of the private AWS S3 bucket
+- `readonly_private_bucket_domain_name` - the domain name of the private AWS S3 bucket
+
+
+- `readonly_public_bucket_name` - the name of the public AWS S3 bucket
```

Comment:
Could you confirm what we mean by `read_only` on all of these please? Obviously concourse is able to write things into the buckets but I'm guessing that only Concourse can write and all other access is read only?

Reply:
Hopefully 47787b0 provides more clarification

Reply:
Yes, I get it. Thanks :)